### DISCUSSION TITLE
black box on my Intel chip Mac

### DISCUSSION THREAD

**Discussion by Suffton** [hacksider/Deep-Live-Cam#446] [General]
conda env, python version is Python 3.10.14
Mac with Intel Core i7
I start it in my conda env and the command is "sudo python run.py"
<img width="1123" alt="截屏2024-08-24 11 09 39" src="https://github.com/user-attachments/assets/ac850c83-a250-4c6a-9702-38cad09d39a4">

I get black box.

**Comment by Suffton** [comment]
pip install onnxruntime==1.16.3 solve this problem. 
onnxruntime-1.19.0 doesn't work

**Reply by VladisanOne** [reply]
That is nice but for the onnxruntime-gpu there seem to only be versions starting from 1.17.0. 
At least that is what pip returns when trying to install the 1.16.3
